Add run_if_changed to postsubmits (#295)
kyma-project/test-infra: docs/prow/migration-guide.md

| **name**                    | The job name. It should clearly identify the job. For reference, see the [convention](./../../prow/README.md#convention-for-naming-jobs) for naming jobs.                                                                                                                                                                                                                                                                                                                             |
| **run_if_changed**          | A regular expression. Define the component for which changes in a PR must trigger the job. If a PR does not modify the component, this job sends a notification to GitHub with the information that it is skipped.                                                                                                                                                                                                                                                                    |

> | **always_run**              | A parameter that defines if a PR must trigger the job automatically. They are mutually exclusive with `run_if_changed`. If neither `run_if_changed` nor `always_run` are set, the job can only be triggered manually by a comment.                                                                         |

Reviewer: ```suggestion
| **always_run**              | A parameter that defines if a PR must trigger the job automatically. `always_run` and `run_if_changed` are mutually exclusive. If neither `run_if_changed` nor `always_run` is set, the job can only be triggered manually by a comment.                                                                         |
```

Reviewer: Modified the sentence mainly to get rid of the passive voice.
